.vib/opensearch/goss/goss.yaml

```diff
@@ -0,0 +1,19 @@
+# Copyright VMware, Inc.
+# SPDX-License-Identifier: APACHE-2.0
+
+file:
```

Comment:
In other solutions we test other elements, see the example of Milvus

```
command:
  {{- $uid := .Vars.proxy.containerSecurityContext.runAsUser }}
  {{- $gid := .Vars.proxy.podSecurityContext.fsGroup }}
  check-user-info:
    # The UID and GID should always be either the one specified as vars (always a bigger number that the def<ault)
    # or the one randomly defined by openshift (larger values). Otherwise, the chart is still using the default value.
    exec: if [ $(id -u) -lt {{ $uid }} ] || [ $(id -G | awk '{print $2}') -lt {{ $gid }} ]; then exit 1; fi
    exit-status: 0
  check-no-capabilities:
    exec: cat /proc/1/status
    exit-status: 0
    stdout:
    - "CapInh:	0000000000000000"
    - "CapPrm:	0000000000000000"
    - "CapEff:	0000000000000000"
    - "CapBnd:	0000000000000000"
    - "CapAmb:	0000000000000000"
  {{ if .Vars.proxy.serviceAccount.automountServiceAccountToken }}
  check-sa:
    exec: cat /var/run/secrets/kubernetes.io/serviceaccount/token | cut -d '.' -f 2 | xargs -I '{}' echo '{}====' | fold -w 4 | sed '$ d' | tr -d '\n' | base64 -d
    exit-status: 0
    stdout:
    - /serviceaccount.*name.*milvus-proxy/
  {{ end }}

```

I would also test that adding a custom configuration gets reflected in opensearch.yml as we do in Milvus. You can use the `config` or `extraConfig` values.

Additionally, I would check that the connection to the rest of Opensearch nodes works and that the metrics exporter works as well

Reply:
[40975bd](https://github.com/bitnami/charts/pull/17509/commits/40975bdc74ba812e486d27712543dea285f8c3f8)